nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -447,9 +483,10 @@ def __init__(self, target_model: NNCFNetwork,
         self._transformation_commands = transformation_commands
         self._filter_importance_fn = filter_importance_fn
         self._weights_normalizer_fn = weights_normalizer_fn
+        self._add_dynamic_inputs = add_dynamic_inputs
 
         graph = self._target_model.get_original_graph()
-        prunable_types = [NNCFConv2d.op_func_name]
+        prunable_types = [NNCFConv2d.op_func_name] #, NNCFLinear.op_func_name]
```

Comment:
Is this you intended? I saw that you added `NNCFLinear.op_func_name` before the latest commit.

Reply:
Yes, I am reverted that until I finish testing a new way to reordering the filters. Thanks!